react-components/src/components/Reveal3DResources/Reveal3DResources.tsx

```diff
@@ -118,14 +118,26 @@ export const Reveal3DResources = ({
         );
       })}
       {image360CollectionAddOptions.map((addModelOption) => {
-        return (
-          <Image360CollectionContainer
-            key={`${addModelOption.siteId}`}
-            siteId={addModelOption.siteId}
-            onLoad={onModelLoaded}
-            onLoadError={onModelLoadedError}
-          />
-        );
+        if ('siteId' in addModelOption) {
```

Comment:
That could be a nice helper function to distinguish between FDM/events based 360 images.